docs/getting-started/automatic-import.asciidoc

```diff
@@ -19,9 +19,9 @@ TIP: Click https://elastic.navattic.com/automatic-import[here] to access an inte
 .Requirements
 [sidebar]
 --
-- A working <<assistant-connect-to-bedrock, Amazon Bedrock connector>>. Automatic Import currently works with all variants of Claude 3. Other models are not supported in this technical preview, but will be supported in future versions.
+- A working <<llm-connector-guides, LLM connector>>. Recommended models: `Claude 3.5 Sonnet`; `GPT-4o`; `Gemini-1.5-pro-002`. 
 - An https://www.elastic.co/pricing[Enterprise] subscription.
-- A sample of the data you want to import, in JSON or NDJSON format. 
+- A sample of the data you want to import, in a structured or unstructured format (including JSON, NDJSON, and Syslog). 
```

Comment:
@jamiehynds with the larger variety of sample data we support, do you think we split the supported formats out into a separate section? Give more space to add details about specific syslog formats, CSV, etc?